content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -396,3 +396,48 @@ Possible errors:
 "The schema of the introspected database was inconsistent: {explanation}"
 
 <!-- Error codes generated by https://github.com/mhwelander/doc-tools/blob/master/error_code_parser.py -->
+
+### Prisma Client (Data Proxy)
+
+#### <inlinecode>P5000</inlinecode>
+
+"This request could not be understood by the server"
+
+#### <inlinecode>P5001</inlinecode>
+
+"This request must be retried"
+
+#### <inlinecode>P5002</inlinecode>
+
+The datasource is an invalid one:
+
+* "Could not parse URL of the datasource"
+* "Datasource URL should use prisma:// protocol"
+* "No valid API key found in the datasource URL"
+
+#### <inlinecode>P5003</inlinecode>
+
+"Requested resource does not exist"
+
+#### <inlinecode>P5004</inlinecode>
+
+The feature is not yet implemented:
+
+* "beforeExit event is not supported yet"
+* "Interactive transactions are not supported yet"
+
```

Comment:
This is a tiny nit, feel free to ignore, but "yet" should precede "supported". ("beforeExit event is not yet supported")